src/register/index.js

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+
+const RegisterComponent = () => {
+  return (
+    <>
+      <div>
+        <h2 className="mt-5 mb-2" style={{ textAlign: "center" }}><b>Create an account</b></h2>
+        <div className="wd-center mb-3">
+          <span className="me-3">I am a</span>
+          <input type="radio" value="BUYER" name="radio-role" id="radio-buyer" className="me-1"/>
+          <label for="radio-buyer" className="me-3"> buyer</label><br/>
+          <input type="radio" value="SELLER" name="radio-role" id="radio-seller" className="me-1"/>
+          <label for="radio-seller" className="me-3"> seller</label><br/>
+        </div>
+      <TextBox placeholder="Username" />
+      <TextBox placeholder="Email" />
+      <TextBox placeholder="Phone number" />
+      <TextBox placeholder="Password" />
```

Comment:
I don't remember how exactly but there's a form style which you can set to password so as you're typing the password it turns into *** (black dots) instead of the letters which might be better. Pretty sure it's just something like <input type="password">

Reply:
ohh cool ill check it out